### DISCUSSION TITLE
Using #Script with a service class

### DISCUSSION THREAD

**Discussion by jonathan-rvezy** [ServiceStack/Discuss#70] [Q&A]
Hi,

I'm not sure if this is a bug, not support, or if I'm doing something wrong.

When I do something like this in my POCO, it works just fine.

```
[AutoPopulate(nameof(IAppUserFilter.AppUserId), Eval = "userSession.UserAuthId.toInt()")]
```

However, if I define a service class to handle the request, it no longer understands what userSession is anymore, it's always empty.

If I do something like

```
[AutoPopulate(nameof(IAppUserFilter.AppUserId), Value = 123)]
```

It's picking up the value as expected in the service class.

**Comment by mythz** [comment]
Please provide minimal code to repro the issue.
